vNext/AISKU/snippet/snippet.js

```diff
@@ -1,96 +1,96 @@
+            localDocument.getElementsByTagName(scriptText)[0].parentNode.appendChild(scriptElement);
+        });
 
-    // capture initial cookie
-    try {
-        appInsights.cookie = localDocument.cookie;
-    } catch (e) { }
+        // capture initial cookie
+        try {
+            appInsights.cookie = localDocument.cookie;
+        } catch (e) { }
 
-    appInsights.queue = [];
-    appInsights.version = 2.0;
+        appInsights.queue = [];
+        appInsights.version = 2.0;
 
-    function createLazyMethod(name) {
-        // Define a temporary method that queues-up a the real method call
-        appInsights[name] = function () {
-            // Capture the original arguments passed to the method
-            var originalArguments = arguments;
-            // Queue-up a call to the real method
-            appInsights.queue.push(function () {
-                // Invoke the real method with the captured original arguments
-                appInsights[name].apply(appInsights, originalArguments);
-            });
-        };
-    }
+        function createLazyMethod(name) {
+            // Define a temporary method that queues-up a the real method call
+            appInsights[name] = function () {
+                // Capture the original arguments passed to the method
+                var originalArguments = arguments;
+                // Queue-up a call to the real method
+                appInsights.queue.push(function () {
+                    // Invoke the real method with the captured original arguments
+                    appInsights[name].apply(appInsights, originalArguments);
+                });
+            };
+        }
 
-    var method = ["Event", "PageView", "Exception", "Trace", "DependencyData", "Metric", "PageViewPerformance"];
-    while (method.length) {
-        createLazyMethod("track" + method.pop());
-    }
+        var method = ["Event", "PageView", "Exception", "Trace", "DependencyData", "Metric", "PageViewPerformance"];
+        while (method.length) {
+            createLazyMethod("track" + method.pop());
+        }
 
-    var track = "Track";
-    var trackPage = track + "Page";
-    createLazyMethod("start" + trackPage);
-    createLazyMethod("stop" + trackPage);
+        var track = "Track";
+        var trackPage = track + "Page";
+        createLazyMethod("start" + trackPage);
+        createLazyMethod("stop" + trackPage);
 
-    var trackEvent = track + method[0];
-    createLazyMethod("start" + trackEvent);
-    createLazyMethod("stop" + trackEvent);
+        var trackEvent = track + method[0];
+        createLazyMethod("start" + trackEvent);
+        createLazyMethod("stop" + trackEvent);
 
-    createLazyMethod("setAuthenticatedUserContext");
-    createLazyMethod("clearAuthenticatedUserContext");
-    createLazyMethod("flush");
+        createLazyMethod("setAuthenticatedUserContext");
+        createLazyMethod("clearAuthenticatedUserContext");
+        createLazyMethod("flush");
```

Comment:
Can you create a lazy method for addTelemetryInitializer as well here? #985

Reply:
yeah I can add this in a separate pr.